src/editor/dialog/build_dialog.cpp

```diff
@@ -0,0 +1,38 @@
+
+#ifdef TOOLS_ENABLED
+
+#include "build_dialog.h"
+
+#include <editor/editor_scale.h>
+#include <modules/kotlin_jvm/src/editor/build/build_manager.h>
+
+BuildDialog::BuildDialog() {
```

Comment:
You should add initialization of `scroll_container` and `log_label` to constructor this way:
```cpp
BuildDialog::BuildDialog() : scroll_container(memnew(ScrollContainer)), log_label(memnew(LogLabel)) {
    ...
}
```